package.json

```diff
@@ -12,7 +12,9 @@
     "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "lint:fix": "eslint --fix 'src/**/*.{ts,tsx}'",
     "preview": "vite preview",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "format": "prettier --config .prettierrc.cjs -c src/**/*",
+    "format:fix": "prettier --config .prettierrc.cjs -w src/**/*"
```

Comment:
Bare et forslag, men siden dere allerede har gått for pre-commit hooks så kan denne kjøres da?

Reply:
Det tester vi :)